io-ballerina/open.bal

```diff
@@ -13,31 +13,30 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
```

Comment:
```suggestion
// specific language governing permissions and limitations
```
Please see above.

Reply:
This is the standard license template we have been using.